subprojects/core/src/main/java/org/gradle/cache/internal/btree/BlockPointer.java

```diff
@@ -15,14 +15,29 @@
  */
 package org.gradle.cache.internal.btree;
 
+import com.google.common.primitives.Longs;
+
 public class BlockPointer implements Comparable<BlockPointer> {
```

Comment:
It seems that this whole class is just a fat `Long`. Can't we just get rid of it and use a good old `long`?